src/main/java/com/example/bookstore/model/Book.java

```diff
@@ -12,19 +12,30 @@
 @Data
 @Entity
 @Table(name = "books")
+
 public class Book {
+
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
     private Long id;
-    @Column(nullable = false)
+    @Column(nullable = false,
+            name = "title")
     private String title;
-    @Column(nullable = false)
+    @Column(name = "author",
+            nullable = false)
     private String author;
-    @Column(nullable = false,
+    @Column(name = "isbn",
+            nullable = false,
             unique = true)
     private String isbn;
-    @Column(nullable = false)
+    @Column(name = "price",
+            nullable = false,
+            columnDefinition = "BIGINT CHECK (price >= 0)"
```

Comment:
for me it seems redundant

Reply:
I wrote this code before learning about input validation, now I know a better way to do it